app/src/main/java/org/systers/mentorship/view/adapters/MembersAdapter.kt

```diff
@@ -1,40 +1,47 @@
 package org.systers.mentorship.view.adapters
 
-import androidx.annotation.NonNull
-import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Filter
+import android.widget.Filterable
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
 import kotlinx.android.synthetic.main.list_member_item.view.*
 import org.systers.mentorship.MentorshipApplication
 import org.systers.mentorship.R
 import org.systers.mentorship.models.User
 import org.systers.mentorship.utils.NON_VALID_VALUE_REPLACEMENT
+import java.util.*
+import kotlin.collections.ArrayList
 
 /**
  * This class represents the adapter that fills in each view of the Members recyclerView
  * @param userList list of users to show
  * @param openDetailFunction function to be called when an item from Members list is clicked
  */
-class MembersAdapter (
-        private val userList: List<User>,
-        private val openDetailFunction: (memberId: Int) -> Unit
-) : RecyclerView.Adapter<MembersAdapter.MembersViewHolder>() {
+class MembersAdapter(
+    private val userList: MutableList<User>,
+    private val openDetailFunction: (memberId: Int) -> Unit
+) : RecyclerView.Adapter<MembersAdapter.MembersViewHolder>(), Filterable {
+
+    private val usersListFull: MutableList<User> = ArrayList(userList)
 
     val context = MentorshipApplication.getContext()
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MembersViewHolder =
-            MembersViewHolder(
-                    LayoutInflater.from(parent.context)
-                            .inflate(R.layout.list_member_item, parent, false)
-            )
+        MembersViewHolder(
+            LayoutInflater.from(parent.context)
+                .inflate(R.layout.list_member_item, parent, false)
+        )
 
     override fun onBindViewHolder(@NonNull holder: MembersViewHolder, position: Int) {
         val item = userList[position]
         val itemView = holder.itemView
 
         itemView.tvName.text = item.name
-        itemView.tvMentorshipAvailability.text = getMentorshipAvailabilityText(item.isAvailableToMentor, item.needsMentoring)
+        itemView.tvMentorshipAvailability.text =
```

Comment:
Can you please remove the extra "enter" between line 43 & 44?

Reply:
It was automatically wrapped like this by Lint (it's longer than 120 chars). But okay, I'll remove it.